.github/workflows/apero-docs-preview.yaml

```
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - gha
+
+name: build-site
+
+jobs:
+  build-site:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      
+      - uses: r-lib/actions/setup-r@v1
+
+      - name: Install dependencies
+        run: |
+          install.packages("blogdown")
+          blogdown::install_hugo("0.79.0", extended=TRUE, force=TRUE)
+          blogdown::install_theme(theme="hugo-apero/hugo-apero", force=TRUE)
+        shell: Rscript {0}
+
+      - name: Build website
+        run: |
+          Rscript -e 'blogdown::build_site(baseURL = "https://hugo-apero.github.io/apero/")'
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
```

```suggestion
        uses: peaceiris/actions-gh-pages@v3.7.3
```

This should be the last version